flake.nix

```
@@ -0,0 +1,41 @@
+{
+  description = "Cubical Agda";
+
+  inputs.flake-utils.url = github:numtide/flake-utils;
+
+  outputs = { self, flake-utils, nixpkgs }:
+    flake-utils.lib.eachDefaultSystem (system:
+    let pkgs = import nixpkgs { inherit system; };
```

```suggestion
    let pkgs = nixpkgs.legacyPackages.${system};
```

https://zimbatm.com/notes/1000-instances-of-nixpkgs

---

I made the modification.

---

Yeah this is fine.